src/views/Conversion.vue

```diff
@@ -171,10 +169,18 @@ export default class Conversion extends Mixins(
         }
 
         // parse firm conversion filing and original business snapshot into store
+        const firmSnapshot = await this.fetchFirmSnapshot()
         await this.parseConversionFiling(changeFiling, firmSnapshot)
       } else {
         // parse business data into store
-        await this.parseEntitySnapshot(firmSnapshot)
+        if (this.isConversionFiling) {
```

Comment:
This will always be True because we are on the conversion filing page (view).

Reply:
Thanks. I'll remove the condition since it is the conversion filing.